Apps/AdminWebClient/src/ClientApp/src/views/Support.vue

```diff
@@ -169,12 +169,21 @@ export default class SupportView extends Vue {
         this.supportService
             .getMessageVerifications(this.selectedQueryType, searchText)
             .then((result) => {
-                this.emailList = result;
+                this.emailList = result.resourcePayload;
+                if (result.resultError) {
+                    debugger;
+                    this.showFeedback = true;
+                    this.bannerFeedback = {
+                        type: FeedbackType.Success,
```

Comment:
Might want to introduce a `FeedbackType.Info` for this and update the BannerFeedback component to handle it.

Reply:
I added an info type and it works, shows up blue.